tests/src/main/java/net/neoforged/neoforge/oldtest/entity/GravityAttributeTest.java

```diff
@@ -60,11 +60,11 @@ public GravityAttributeTest(IEventBus modEventBus) {
     }
 
     @SubscribeEvent
-    public void worldTick(TickEvent.LevelTickEvent event) {
-        if (!event.level.isClientSide) {
+    public void worldTick(LevelTickEvent.Post event) {
+        if (!event.getLevel().isClientSide()) {
```

Comment:
I think an inline instance of: ` (event.getLevel() instanceof ServerLevel serverLevel)` would be a that bit better.